scripts/QMCUtils.py

```diff
@@ -1486,4 +1492,76 @@ def get_nevptEnergy(fname="nevpt2.out",printNevpt2=False):
     totalE = nevptE + sum(icE)
     return totalE,Error
   
+def prepAFQMC_fromFCIDUMP(choleskyThreshold=2.e-3,ndets=100,left=None,right=None,seed = None,spin=None,norb_core=0,norb_act=None,nelec_act=None,norb_frozen=None,nroot=0,mo_coeff=None,chol_cut = 1e-5,dt = 0.005, steps_per_block = 50, nwalk_per_proc = 5, nblocks = 1000, ortho_steps = 20, cholesky_threshold = 2.0e-3,fname="afqmc.json",fcidump="FCIDUMP"): 
+    print("Calculating Cholesky integrals")
+    fcidump = tools.fcidump.read(filename=fcidump,molpro_orbsym=True)
```

Comment:
this is a pyscf function which is fine here but on its own, you don't want to use it
you can look at some of the uihf or gihf functions which pretty much only take integrals as input